File: LV1/1.4/heartrate/heartrate.go
package main

import "fmt"

func main() {
	n := 0
	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		var b, p float64
		fmt.Scanf("%f %f", &b, &p)

		bpm := 60 * b / p
		abpm := 60 / p
		fmt.Println(bpm-abpm, bpm, bpm+abpm)
	}
}

File: LV1/1.4/heartrate/heartrate.rb
n = gets.to_i

n.times do
  line = gets.split
  b = line[0].to_i
  p = line[1].to_f

  bpm = 60 * b / p
  abpm = 60 / p
  puts "#{bpm - abpm} #{bpm} #{bpm + abpm}"
end
